datastream-kafka/src/main/java/com/linkedin/datastream/kafka/EmbeddedKafkaCluster.java

```diff
@@ -179,7 +180,7 @@ public String toString() {
     return sb.toString();
   }
 
-  static class SystemTime implements Time {
+  static class EmbeddedSystemTime extends SystemTime implements Time {
```

Comment:
What is the reason behind this change?

Reply:
SystemTime is introduced as an interface into kafka lib. So we need to create a different one to override the behaviour.